acl/src/test/java/org/springframework/security/acls/jdbc/AclClassIdUtilsTest.java

```diff
@@ -157,4 +154,15 @@ public void shouldReturnUUIDWhenUUIDClassIdType() throws SQLException {
 		assertThat(newIdentifier).isEqualTo(identifier);
 	}
 
+	@Test
+	public void shouldReturnStringWhenStringClassIdType() throws SQLException {
+		// given
+		given(resultSet.getString("class_id_type")).willReturn("java.lang.String");
+
+		// when
+		Serializable newIdentifier = aclClassIdUtils.identifierFrom(IDENTIFIER_AS_STRING, resultSet);
```

Comment:
This extra variable seems needless. Could you please use `DEFAULT_IDENTIFIER_AS_STRING` instead?